### PR TITLE
Add front camera support for android

### PR DESCRIPTION
**Pull request by olehmisuna**
In order to use front camera, add following to `app\src\main\AndroidManifest.xml` in your project.

```
<uses-permission android:name="android.permission.CAMERA" />
<uses-feature android:name="android.hardware.camera" android:required="false" />
<uses-feature android:name="android.hardware.camera.front" android:required="false" />
```

### CODE REVIEW

**Comment by ivpusic** [comment]
could you also update readme file with necessary permissions for this feature?

**Comment by olehmisuna** [comment]
Sure.

**Comment by olehmisuna** [comment]
> could you also update readme file with necessary permissions for this feature?

Sure, updated.

**Comment by ivpusic** [comment]
thanks!

File: android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModule.java [hunk 1]
```diff
@@ -305,6 +307,10 @@ private void initiateCamera(Activity activity) {
 
             cameraIntent.putExtra(MediaStore.EXTRA_OUTPUT, mCameraCaptureURI);
 
+            if (this.useFrontCamera) {
+                cameraIntent.putExtra("android.intent.extras.CAMERA_FACING", 1);
```
Comment (juanjonucleus) [inline android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModule.java]: You also need to add these two lines, this flag doesn't work on the Google Pixel 
cameraIntent.putExtra("android.intent.extras.LENS_FACING_FRONT", 1);
cameraIntent.putExtra("android.intent.extra.USE_FRONT_CAMERA", true);

Comment (ivpusic) [reply android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModule.java]: @juanjonucleus could you send a PR with this fix?

**Pull request merged**